src/metpy/plots/declarative.py

```diff
@@ -786,6 +786,15 @@ def ax(self):
 
         return self._ax
 
+    @property
+    def plot_kwargs(self):
```

Comment:
Can you add this to `Panel`:
```python
@property
def plot_kwargs(self):
    return {}
```
Otherwise, this is definitely going to fail straight away when you try to plot with non-maps.

Reply:
Sure thing. I actually had that in there as I was working on things, but took it out since it wouldn't have been covered by a test.

Reply:
Sure. I just figure since there are other things that aren't covered, might as well get more things that are complete--since they're not actually able to be exercised by users yet.